package.json

```
@@ -3,18 +3,18 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "start": "serve -s ./build",
+    "start": "react-scripts start",
```

@luc-tuyishime, would you please have the start script run the build
```suggestion
    "start": "serve -s ./build",
```

---

ok